routes/alumni.js

```diff
@@ -14,4 +14,16 @@ router.get('/', function(req, res, next) {
   })
 });
 
+router.post('/', function(req, res, next) {
+  var newAlumni = new alumni(req.body.newAlumni);
+  newAlumni.save(function(err, results){
+      if(err) {
+          console.log(req.body);
+          res.status(404);
+      } else {
+          res.status(201).send('Alumni Added')
```

Comment:
it is often useful to send back the item that was created, for example so the person that posted can know the id that was created in the db